scripts/hub-stress-test.py

```diff
@@ -519,7 +527,7 @@ def run_stress_test(count, batch_size, token, endpoint, dry_run=False,
     users = create_users(count, batch_size, endpoint, session,
                          existing_users=existing_users)
     # Now that we've created the users, start a server for each in batches.
-    start_servers(users, endpoint, session)
+    start_servers(users, endpoint, session, profile)
```

Comment:
```suggestion
    start_servers(users, endpoint, session, profile=profile)
```

nit but would be good to pass as a kwarg since then if the signature changes in the future we don't have to worry about arg location